source/includes/_json_single_petition.md

```diff
@@ -45,6 +44,22 @@ $(document).ready(function(){
   "image_url": "https://d8s293fyljwh4.cloudfront.net/petitions/images/92283/original/2016-06-20-1466458252-1098096-ywllowbrickroad.jpg?1473884741",
   "creator_name": "Kristyn Arrighi",
   "signature_count": 223
+  "location": {
+    "country":"US",
+    "latitude":"42.9399637",
+    "locality":"Buffalo",
+    "longitude":"-78.8087571",
+    "postal_code":"",
+    "query":"Land of Oz, Kensington Avenue, Buffalo, NY, United States",
+    "region":"NY",
+    "street":"Kensington Ave",
+    "street_address":" Kensington Ave",
```

Comment:
This is realistic in that it looks a whole lot like real data -- but should we use an example where the `street_number` is set and therefore the `street_address` doesn't start with a space?